ui/app/templates/components/oidc/key-form.hbs

```diff
@@ -51,19 +51,23 @@
         @value="limited"
         @groupValue={{this.radioCardGroupValue}}
         @onChange={{this.handleClientSelection}}
+        @disabled={{@model.isNew}}
+        @disabledTooltipMessage="This option has been disabled for now. To limit access, you must first create an application that references this key."
```

Comment:
I know that the "to limit access" text refers to the text on the radio card box, but it reads oddly. The words limit access to me —at first— read, to limit someones access. Maybe check with design? Something along the lines of, "To enable this option, you must first..." 

Let me know if that makes sense why it's confusing.

Reply:
I pulled this from figma - but will touch base with @yankun91  when she's back!

Reply:
@hellobontempo @Monkeychip thanks for checking here! Maybe I am too familiar with the terms already, so I felt "limit access" is kinda straightforward to me that it's limiting to the key. An older version of my attempt reads like this: "This option has been disabled for now and will be available once you create an application that references this key."

Reply:
I think both work! I agree that 'limit access' is clear to me, but maybe I'm also too familiar with the language!